zulipterminal/config/keys.py

```diff
@@ -151,6 +151,7 @@
     ('ENTER', {
         'keys': {'enter'},
         'help_text': 'Perform current action',
+        'user_state': ''
```

Comment:
I don't think they should be different in different sections. It could be confusing for users.

About changing the text, do you have anything in mind?

Reply:
Just from exploring the UI (this can be confirmed via the source):
* enter on a button, changes the view to that 'narrow'
* enter on a message, does a 'contextual reply' (in that narrow?)
* enter on a stream/message/user search box (runs the search &) moves to the results

Reply:
Just to clarify, you're suggesting that we have 3 'ENTER' keys in the dictionary with different `help _text`?

Reply:
We could give help tips suggestions for 'ENTER' based on where the current focus is, i.e whether it is on streamButton, SearchBox, etc (v2 perhaps?)